Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,7 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+function calc(weigh,height){
+    const bmi = weigh/(height * height);
+   return bmi.toFixed(1)
```

Comment:
What is the **type** of value you expect to be returned from this function?  A number or a string? Does your function return the type of value you expected?

Reply:
i added    return  parseFloat(bmi.toFixed(1)) now will return a number